projects_test.go

```diff
@@ -85,7 +85,8 @@ func TestProjectsService_GetProject_WithSlash(t *testing.T) {
 		testMethod(t, r, "GET")
 		testRequestURL(t, r, "/projects/plugins%2Fdelete-project")
 
-		fmt.Fprint(w, `)]}'`+"\n"+`{"id":"plugins%2Fdelete-project","name":"plugins/delete-project","parent":"Public-Plugins","description":"A plugin which allows projects to be deleted from Gerrit via an SSH command","state":"ACTIVE"}`)
+		c := `)]}'` + "\n" + `{"id":"plugins%2Fdelete-project","name":"plugins/delete-project","parent":"Public-Plugins","description":"A plugin which allows projects to be deleted from Gerrit via an SSH command","state":"ACTIVE"}`
+		fmt.Fprint(w, c)
```

Comment:
FWIW, this is fine, and you can also consider using `io.WriteString(w, "... some string ...")` in a situation like this to avoid warnings about possible formatting directives, since there's no formatting being used here.

Reply:
Thanks @dmitshur - Today I Learned :)
For now, I keep it like this